packages/neuron-wallet/src/services/live-cells.ts

```diff
@@ -0,0 +1,50 @@
+import { getConnection, In } from 'typeorm'
+import { hexToBytes } from '@nervosnetwork/ckb-sdk-utils'
+import LiveCellEntity from 'database/chain/entities/live-cell'
+
+export default class LiveCellsService {
+  // exclude hasData = true and typeScript != null
+  public static async getBalance(lockHashes: Set<string>): Promise<{
+    liveBalance: Map<string, string>
+    sentBalance: Map<string, string>
+    pendingBalance: Map<string, string>
+  }> {
+    const hashData: Buffer[] = []
+    lockHashes.forEach((lockHash: string) => hashData.push(Buffer.from(hexToBytes(lockHash))))
```

Comment:
`Buffer.from(hexToBytes(lockHash))` could be `Buffer.from(lockhash.slice(2), 'hex')`